project.clj

```diff
@@ -96,6 +98,15 @@
   :lein-release {:scm        :git
                  :deploy-via :lein-deploy}
 
+  :uberjar-name "puppetdb-release.jar"
+  :lein-ezbake {:vars {:user "puppetdb"
+                       :group "puppetdb"
+                       :build-type "foss"
```

Comment:
I noticed that the only difference between `puppetdb` and `pe-puppetdb` is that `:build-type` is `pe` and the user/group names have the `pe-` prefix. I'm thinking about ways to accomodate this that aren't totally janky:

* We could take arbitrary key-value pairs on at the end of the `lein ezbake <action>` command line that would shadow the values set in this group of vars. 
* We could use `pe` and `foss` leiningen profiles that contain different `:lein-ezbake` values. Then the command to build pe-puppetdb would be `lein with-profile ezbake,pe ezbake build` as opposed to `lein with-profile ezbake ezbake build`.